tools/extras/install_mmseg.sh

```diff
@@ -39,29 +39,47 @@ if [ -d ./mmseg-1.3.0 ] ; then
   echo  >&2 "$0: Warning: old installation of mmseg found. You should manually"
   echo  >&2 "  delete the directory tools/mmseg and "
   echo  >&2 "  edit the file tools/env.sh and remove manually all references to it"
+  exit 1
 fi
 
 if [ ! -d ./mmseg-1.3.0 ] ; then
   wget http://pypi.python.org/packages/source/m/mmseg/mmseg-1.3.0.tar.gz
   tar xf mmseg-1.3.0.tar.gz
 fi
 
-pyver=`python --version 2>&1 | sed -e 's:.*\([2-3]\.[0-9]\+\).*:\1:g'`
-export PYTHONPATH=$PYTHONPATH:`pwd`/mmseg-1.3.0/lib/python${pyver}/site-packages
+(
 cd mmseg-1.3.0
-mkdir -p lib/python${pyver}/site-packages
+pyver=`python --version 2>&1 | sed -e 's:.*\([2-3]\.[0-9]\+\).*:\1:g'`
+export PYTHONPATH=$PYTHONPATH:$PWD/lib/python${pyver}/site-packages/:$PWD/lib64/python${pyver}/site-packages/
+# we have to create those dir, as the install target does not create it
+mkdir -p $PWD/lib/python${pyver}/site-packages/
+mkdir -p $PWD/lib64/python${pyver}/site-packages/
 python setup.py build
 python setup.py install --prefix `pwd`
-cd ../
-
-(
-  set +u
-  pyver=`python --version 2>&1 | sed -e 's:.*\([2-3]\.[0-9]\+\).*:\1:g'`
-  wd=`pwd`
+)
 
-  [ -f ./env.sh ] && . ./env.sh
+## we first find the mmseg.py file (the module name which will be imported,
+## so that should be pretty reliable) and then we work out the location of
+## the site-packages directory (typically it would be one level up from
+## the location of the mmseg.py file but using find seems more reliable
+mmseg_file_lib=$(find ./mmseg-1.3.0/lib/ -type f -name mmseg.py | head -n 1)
+mmseg_file_lib64=$(find ./mmseg-1.3.0/lib64/ -type f -name mmseg.py | head -n 1)
+if [ -f $mmseg_file_lib ]; then
```

Comment:
For some reason, this check is failing on my machine: The $mmseg_file_lib variable is empty, but despite that, this condition is being triggered.

How about this:

```
if [ "x${mmseg_file_lib}" != "x" ]; then
  lib_dir=./lib/
elif [ "x${mmseg_file_lib64}" != "x" ]; then
  lib_dir=./lib64
fi
```

The above worked for me, although, as far as I am aware, there seems to be no reason why the one you have proposed should not work